sormas-backend/src/test/java/de/symeda/sormas/backend/caze/CaseFacadeEjbTest.java

```diff
@@ -883,6 +886,66 @@ public void testGetIndexListByARestrictedAccessToAssignedEntities() {
 		assertEquals(1, results2.size());
 	}
 
+	@Test
+	public void testGetIndexListWithLimitedDisease() {
+
+		String lastName = "Person";
+		PersonDto cazePerson1 = creator.createPerson("Case1", lastName);
+		final CaseDataDto case1 = creator.createCase(
+			surveillanceOfficer.toReference(),
+			cazePerson1.toReference(),
+			Disease.EVD,
+			CaseClassification.PROBABLE,
+			InvestigationStatus.PENDING,
+			new Date(),
+			rdcf);
+
+		PersonDto cazePerson2 = creator.createPerson("Case2", lastName);
+		final CaseDataDto case2 = creator.createCase(
+			surveillanceOfficer.toReference(),
+			cazePerson2.toReference(),
+			Disease.CORONAVIRUS,
+			CaseClassification.PROBABLE,
+			InvestigationStatus.PENDING,
+			new Date(),
+			rdcf);
+
+		PersonDto cazePerson3 = creator.createPerson("Case3", lastName);
+		final CaseDataDto case3 = creator.createCase(
+			surveillanceOfficer.toReference(),
+			cazePerson3.toReference(),
+			Disease.EVD,
+			CaseClassification.PROBABLE,
+			InvestigationStatus.PENDING,
+			new Date(),
+			rdcf1);
+
+		PersonDto cazePerson4 = creator.createPerson("Case2", lastName);
+		final CaseDataDto case4 = creator.createCase(
+			surveillanceOfficer.toReference(),
+			cazePerson4.toReference(),
+			Disease.CORONAVIRUS,
+			CaseClassification.PROBABLE,
+			InvestigationStatus.PENDING,
+			new Date(),
+			rdcf1);
+
+		loginWith(nationalAdmin);
+		assertEquals(4, getCaseFacade().getIndexList(null, 0, 100, null).size());
```

Comment:
there is no need to assert that for national admin the get index list works, it is already tested in other tests

Reply:
code removed